src/my_project/utils/cover_processing.py

```
@@ -151,6 +147,6 @@ def run(self):
         try:
             self.extract_image_bytes()
             self.save_cover()
-            logging.info("Cover saved!")
+            logger.info("Cover saved!")
         except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)